.gitignore

```diff
@@ -107,3 +107,6 @@ gitleaks.log
 *.jtl
 
 terraform-fmt.log
+
+# CastleMock event files
+.environment/soap_service/soap/event/**
```

Comment:
Telling GitHub to ignore all the event files that are created by CastleMock. Each request to CastleMock will create a file in this folder. It gets noisy.